src/v/cluster/partition_probe.cc

```diff
@@ -267,6 +267,142 @@ void replicated_partition_probe::setup_public_metrics(const model::ntp& ntp) {
           });
     }
+
+    setup_public_scrubber_metric(ntp);
+}
+
+void replicated_partition_probe::setup_public_scrubber_metric(
+  const model::ntp& ntp) {
+    namespace sm = ss::metrics;
+
+    // No point in setting up the scrubber metrics if there's no
+    // archival metadata STM to pull values from.
+    if (!_partition.archival_meta_stm()) {
+        return;
+    }
+
+    auto ns_label = metrics::make_namespaced_label("namespace");
+    auto topic_label = metrics::make_namespaced_label("topic");
+    auto partition_label = metrics::make_namespaced_label("partition");
+
+    const std::vector<sm::label_instance> common_labels{
+      ns_label(ntp.ns()),
+      topic_label(ntp.tp.topic()),
+      partition_label(ntp.tp.partition()),
+    };
+
+    auto anomaly_type_label = metrics::make_namespaced_label("type");
+    auto severity_type_label = metrics::make_namespaced_label("severity");
+
+    auto generate = [&, this](
+                      ss::sstring name,
+                      ss::sstring description,
+                      ss::sstring severity,
+                      auto extractor) -> ss::metrics::metric_definition {
+        auto labels = common_labels;
+        labels.push_back(anomaly_type_label(std::move(name)));
+        labels.push_back(severity_type_label(std::move(severity)));
+
+        return sm::make_gauge(
+                 "anomalies",
+                 [this, extractor = std::move(extractor)] {
+                     if (
+                       !_partition.is_elected_leader()
+                       || !_partition.archival_meta_stm()) {
+                         return size_t{0};
+                     }
+
+                     const auto& anomalies = _partition.archival_meta_stm()
+                                               ->manifest()
+                                               .detected_anomalies();
+
+                     return extractor(anomalies);
+                 },
+                 sm::description(std::move(description)),
+                 labels)
+          .aggregate({sm::shard_label, partition_label});
+    };
+
+    _public_metrics.add_group(
+      prometheus_sanitize::metrics_name("cloud_storage"),
+      {
+        generate(
+          "missing_partition_manifest",
+          "Count of missing partition manifest anomalies for the topic",
+          "high",
+          [](const cloud_storage::anomalies& anomalies) {
+              return anomalies.missing_partition_manifest ? size_t{1}
+                                                          : size_t{0};
+          }),
+        generate(
+          "missing_segments",
+          "Count of segments referenced by metadata which are not present in "
+          "cloud storage",
+          "high",
+          [](const cloud_storage::anomalies& anomalies) {
+              return anomalies.missing_segments.size();
+          }),
+        generate(
+          "missing_spillover_manifests",
```

Comment:
@VladLazar why does it matter to an admin whether its spillover manifests or any other kind of manifest?

Reply:
It doesn't matter to the admin, but it's a quality of life feature for the developer fixing the issue. Imagine we aggregated everything under a single anomaly metric. Once something was fixed, it would be impossible to tell if all instances of that issue have gone away or not.